src/main/java/org/nastya/entity/Book.java

```diff
@@ -0,0 +1,51 @@
+package org.nastya.entity;
+
+public class Book {
+
+    private int id;
+    private String title;
+    private int published;
```

Comment:
"published" is a bad name for the field. It is not clear is it a flag that means whether it has been published or not.
Rename to publishingYear.